apps/docs/pages/_app.jsx

```diff
@@ -5,7 +5,8 @@ import componentData from "../data/componentData";
 
 import "./styles.css";
 import "./normalize.css";
-import "@productplan/atlas/dist/index.cjs.css";
+import "@productplan/atlas-tokens/dist/styles/tokens.css";
+import "@productplan/atlas/dist/atlas.css";
```

Comment:
Even if this looks worse than before, I think this works better than before:

- This way, all the `token` are generated in parallel and are not part of the `atlas-react` package.  Therefore, we now have the ability to use the variables even outside our components.  We don't have to export the whole `atlas.css` in order to get the `tokens.css` only, for example.
- Themeability. In order to create different colors, we now only need to create different values for the CSS variables.
- If we are planning to do `web-components`, we still need a way to generate those tokens in parallel and consume them from the upper level.  Why? Because we don't want to inject several css rules to the `:host`.